completions.go

```diff
@@ -270,6 +270,14 @@ func (c *Command) initCompleteCmd(args []string) {
 	}
 }
 
+// sliceValue is a reduced version of [pflag.SliceValue]. It is used to detect
+// flags that accept multiple values and therefore can provide completion
+// multiple times.
+type sliceValue interface {
```

Comment:
I think that if we want to allow programs to do
`var _ sliceValue = (*customMultiString)(nil)` like in the tests, we need to export the type.